docs/usage/install.rst

```diff
@@ -1,3 +1,230 @@
-Installation
-============
+Installation Instructions
+#########################
+
+PlaidML supports :ref:`Ubuntu`, :ref:`macOS` , and :ref:`Microsoft Windows` operating systems.
+
+Ubuntu
+******
+
+If necessary, install Python's ``pip`` tool. OpenCL 1.2 or greater is also
+
+required.
+
+.. code-block::
+
+    sudo add-apt-repository universe && sudo apt update
+
+    sudo apt install python3-pip
+
+    sudo apt install clinfo
+    
+Run ``clinfo``, and if it reports ``"Number of platforms" == 0``, you
+
+can install a driver (GPU) or enable a CPU via one of these options:
+
+* **Nvidia** -- For Nvidia GPUs, run:
+
+.. code-block::
+
+    sudo add-apt-repository ppa:graphics-drivers/ppa && sudo apt update
+
+    sudo apt install nvidia-modprobe nvidia-384 nvidia-opencl-icd-384 libcuda1-384
+
+
+
+* **AMD** -- For AMD graphics cards, `download the AMDGPU PRO driver <http://support.amd.com/en-us/kb-articles/Pages/AMDGPU-PRO-Driver-for-Linux-Release-Notes.aspx>`_ and follow
+
+the instructions provided by AMD for the chip.
+
+* **Intel® Xeon® Processors OR Intel® Core™ Processors** -- In lieu of
+
+installing specific drivers, you can [install nGraph with pip], or you can
+
+[build the nGraph Library] with the cmake flag `-DNGRAPH*PLAIDML*ENABLE=TRUE`.
+
+
+Python
+======
+
+Although PlaidML can be run with Python2, we recommend Python3, as well as
+
+judicious use of a `Virtualenv <https://virtualenv.pypa.io/en/stable>`_.  To create one just for using PlaidML:
+
+.. code-block::
+
+    python3 -m venv plaidml-venv
+
+    source plaidml-venv/bin/activate
+
+
+Keras
+=====
+
+There are two ways to get Keras working on your system:
+
+1. Isolate it to your `venv` as follows:
+
+.. code-block:: shell
+
+    pip install -U plaidml-keras
+
+
+2. Alternatively, install the PlaidML wheels system-wide with:
+
+.. code-block::
+
+    sudo -H pip install -U plaidml-keras
+
+
+
+Finally, set up PlaidML to use a preferred computing device:
+
+.. code-block::
+
+    plaidml-setup
+
+
+
+You can test the installation by running MobileNet in `plaidbench <https://github.com/plaidml/plaidbench>`_. Remember to
```

Comment:
The `pkaidbench` repo has been archived and the contents are now a part of the `plaidml` repo. We should get rid of the link to the old `plaidbench` repo.